Resources/Prototypes/_Moffstation/PersonalItems/Wearables/SapphireMaris/Sapphire_Maris/sapphireampule.yml

```
@@ -0,0 +1,26 @@
+# Player: SapphireMaris - Character: Sapphire Maris
+- type: entity
+  parent: ClothingNeckBase
+  id: PersonalItemSapphireAmpule
+  name: crystal ampule
+  description: A vial etched with the words “Remember Me” contains a swirling blue liquid, its surface catching the light like liquid crystal.
```

Description needs to be grammatically correct. A possible suggestion is:
```suggestion
  description: An ampule containing a swirling blue liquid, its surface catching the light like liquid crystal. Scratched on the ampule are the words "Remember me".
```